webapp/src/shared/components/Header/Header.tsx

```diff
@@ -0,0 +1,51 @@
+import { Link } from 'react-router-dom';
+import { Color } from '../../types';
+import { SmallAvatar } from '../Avatar/Avatar';
+import Icon, { IconSize, IconVariant } from '../Icon/Icon';
+import Status, { StatusVariant } from '../Status/Status';
+import Text, { TextColor, TextVariant, TextWeight } from '../Text/Text';
+import './Header.css';
+
+type HeaderProps = {
+  navigationFigure: string;
```

Comment:
Nice!! but I'd explicitly add `IconVariant` so it is more clear what options can we pass around

```suggestion
  navigationFigure: IconVariant | string;
```